docker/services/fcrepo/Dockerfile

```diff
@@ -5,8 +5,8 @@ MAINTAINER Yinlin Chen "ylchen@vt.edu"
 RUN apt-get update && apt-get -y upgrade
 
 # Install Fedora4
-ARG FEDORA_VERSION=4.7.5
-ARG FEDORA_TAG=4.7.5
+ARG FEDORA_VERSION=5.0.0
```

Comment:
Should this also be:
```
${FEDORA_VERSION}
${FEDORA_TAG}
```

Reply:
@awoods good catch! I remove these since the version number will be passed through docker-compose.